security/pfSense-pkg-HoneyTrap/Makefile

```diff
@@ -0,0 +1,53 @@
+# $FreeBSD$
+
+PORTNAME=	pfSense-pkg-HoneyTrap
+PORTVERSION=	0.1
+CATEGORIES=	security
+MASTER_SITES=	# empty
+DISTFILES=	# empty
+EXTRACT_ONLY=	# empty
+
+MAINTAINER=	remco.verhoef@dutchsec.com
+COMMENT=	PfSense package HoneyTrap
+
+LICENSE=	ART20
+
+RUN_DEPENDS=	honeytrap:security/honeytrap
```

Comment:
Use `honeytrap>0:security/honeytrap` to make ports infrastructure consider dependency of a package and not a file